### DISCUSSION TITLE
Tolerations for jobs

### DISCUSSION THREAD

**Discussion by lfauchille** [aquasecurity/starboard#507] [Development]
Hey 👋 

My Kubernetes cluster does not contain untaint nodegroups.
I installed starboard using krew, and launched this command : 

```sh
kubectl starboard scan vulnerabilityreports deployments/nginx
```

And the job never launched because of `0/x nodes are available: x node(s) had taint {stackName: true}, that the pod didn't tolerate `
I'd be great then to edit Operator or Command-line configuration to add tolerations

**Environment:**

- Starboard version : 
```
Starboard Version: {Version:0.10.0 Commit:40f061c9721476091d080e27d191a3c9c3182bcf Date:2021-04-13T20:51:53Z} 
```

Thanks a lot for all the work you did !

**Comment by danielpacak** [comment]
👋 @lfauchille Thanks for the feedback. I think it makes perfect sense to add config for tolerations, e.g. add it to the `starboard` ConfigMap and read it from there whenever we create a K8s Job.

**Reply by danielpacak** [reply]
I have created #538 to follow up on this requirement.
